lib/fuel-soap.js

```diff
@@ -328,8 +328,26 @@ FuelSoap.prototype.describe = function (type, callback) {
 	}, callback);
 };
 
-FuelSoap.prototype.execute = function () {
+FuelSoap.prototype.execute = function (type, props, callback) {
 
+  var body = {
+    ExecuteRequestMsg: {
+			'$': {
+				'xmlns': 'http://exacttarget.com/wsdl/partnerAPI'
+			},
+      Requests: {
+        Name: type,
+        Parameters: props
```

Comment:
I know this sounds kind of nitpicky, but why not params instead of props? It just seems to make better since.